packages/account/src/Components/additional-kyc-info-modal/form-config.tsx

```diff
@@ -0,0 +1,308 @@
+import React from 'react';
+import { GetSettings, ResidenceList } from '@deriv/api-types';
+import { Localize, localize } from '@deriv/translations';
+import { generateValidationFunction } from '@deriv/shared';
+
+// Type for the list of items in a dropdown or select
+export type TListItem = {
+    /**
+     * The text of the item (e.g. 'United Kingdom', 'Germany', etc.)
+     */
+    text: string;
+    /**
+     * The value of the item
+     */
+    value: string;
+};
+
+// Type for the 'name' of the field
+export type TFields = 'place_of_birth' | 'tax_residence' | 'tax_identification_number' | 'account_opening_reason';
+
+// Type for the 'req' rule
+type ReqRule = ['req', React.ReactNode];
+
+// Type for the 'length' rule
+type LengthRule = ['length', React.ReactNode, { min: number; max: number }];
+
+// Type for the 'regular' rule
+type RegularRule = ['regular', React.ReactNode, { regex: RegExp }];
+
+// Type for a custom validation function
+type CustomValidator = (
+    /**
+     * The value of the field
```

Comment:
Certain comments are unnecessary, please only add comments when they are needed.
I understand that they may show up when you hover on the variables but they are not helpful.

Reply:
Sure thing @matin-deriv 
Updated